### PR TITLE
Feature/about_translation

### PR DESCRIPTION
**Pull request by noctillion**
Description:
This PR adds support for translating the `about.html` content in the Overview tab. The changes introduced include:

Changes:
- Basic templates eng_about.html and fr_about.html
- Fetching both English and French versions of `x_about.html` content using async thunk.
- Dynamically rendering the `x_about.html` content in the Overview tab based on the active language.

### CODE REVIEW

File: src/js/constants/contentConstants.js [hunk 1]
```diff
@@ -1 +1 @@
-export const aboutUrl = '/public/about.html';
+export const aboutUrl = '/public/';
```
Comment (brouillette) [inline src/js/constants/contentConstants.js:1]: Nitpick: 'might be worth renaming `aboutUrl` to `partialAboutUrl` for consistency

Comment (noctillion) [reply src/js/constants/contentConstants.js]: Good idea!

**Comment by noctillion** [comment]
Thanks for the review!

**Review by SanjeevLakhwani — CHANGES_REQUESTED** [review]
I would suggest using useTranslation hook for it as it would abstract the language change and a requirement for a french file, which is currently being called on initial access. 

If working with a work around, other changes would be to use two constants for en and fr files.

Also, this would cause a merge conflict as bento-public is now in typescript

**Comment by noctillion** [comment]
Closed to change to PR- Feature/about translation ts #76

**Pull request closed without merging**